group2_assignment2_solution.ipynb

```diff
@@ -495,7 +507,10 @@
     "you have crawled for each keyword, save them into a file named w.json where w is the keyword.\n",
     "'''\n",
     "# saving the tweets in three json files, one for each keyword\n",
-    "#INSERT YOUR CODE HERE"
+    "#INSERT YOUR CODE HERE\n",
+    "save_to_json(k1_tweets, 'group2_computer.json')\n",
+    "save_to_json(k2_tweets, 'group2_animal.json')\n",
+    "save_to_json(k3_tweets, 'group2_movie.json')"
```

Comment:
I really wanted to use a loop here to reduce repetition, but we are stuck with the variables `k1_tweets` etc. which are annoyingly separated from the data. The name and retrieved values should all probably be stored in single object. I am probably overthinking it:

```
for index, keyword in enumerate(keywords):
    save_to_json(data_goes_here, "group2_{keyword}.json")
```